.template/addons/github/.github/wiki/_Sidebar.md

```diff
@@ -0,0 +1,12 @@
+## Table of Contents
+
+- [[Home]]
+- [[Getting Started]]
+
```

Comment:
I recommend adding another section for "Architecture" as per [the template on Compass](https://nimblehq.co/compass/development/documentation/github-wiki/#organization). Under `architecture`, we can add one default page for authentication as most apps need it.

Reply:
Added the `Architecture` section ✨  - [342250e](https://github.com/nimblehq/rails-templates/pull/317/commits/342250e1afd3459a82f4fee8158d29c2542b7382)